packages/dds/tree/src/test/simple-tree/api/schemaCreationUtilities.spec.ts

```diff
@@ -102,6 +149,38 @@ describe("schemaCreationUtilities", () => {
 		}
 	});
 
+	it("adaptEnum - construction tests", () => {
```

Comment:
We currently only support string-based enums, right? What error mode does the user see if they try to use a number-based enum?

Reply:
I tested it and apparently there is no error. I have added tests and indicate they are supported now.